ygen/gogen.go

```diff
@@ -34,6 +34,20 @@ const (
 	// by specifying a name within the GoOptions field of the YANGCodeGenerator
 	// instance used for code generation.
 	defaultSchemaVarName string = "ySchema"
+	// defaultPackageName specifies the default name that should be
+	// used for the generated Go package.
+	defaultPackageName string = "ocstructs"
+)
+
+var (
+	// packageImportPaths stores the default import paths that should be used
+	// within the generated Go. It is keyed by a short name for a particular
+	// import.
+	packageImportPaths = map[string]string{
+		"ygot":   "github.com/openconfig/ygot/ygot",
+		"goyang": "github.com/openconfig/goyang/pkg/yang",
+		"ytypes": "github.com/openconfig/ygot/ytypes",
+	}
```

Comment:
They were, see the comment above for why this was the case. Essentially, the test cases within the library fail if we just have them in `generator.go`.

I refactored this such that we have them in `gogen.go` as constants that are referred to in `generator.go` such that the user of that binary knows what the import paths are expected to be by default. PTAL.